doc/changelog.rst

```diff
@@ -34,6 +35,9 @@ Description
 
 Detailed Notes
 
+- The dashboard needs to display historical logs, so log files are written
+  out under the .smartsim directory and symlinked to the files in the 
```

Comment:
Pedantic, but... aren't we actually symlinking the ones in the experiment directory _to_ the ones under `.smartsim`?